File: IHIS/IHIS.Framework/IHIS.CloudConnector/Contracts/Models/Drgs/DRG0120U00Grd0120Item3Info.cs
using System;

namespace IHIS.CloudConnector.Contracts.Models.Drgs
{
    [Serializable]
	public class DRG0120U00Grd0120Item3Info
	{
		private String _bunryu1Name;
		private String _bogyongCode;
		private String _bogyongName;
		private String _dv;
		private String _banghyang;
		private String _buwi;
		private String _chiryo;

		public String Bunryu1Name
		{
			get { return this._bunryu1Name; }
			set { this._bunryu1Name = value; }
		}

		public String BogyongCode
		{
			get { return this._bogyongCode; }
			set { this._bogyongCode = value; }
		}

		public String BogyongName
		{
			get { return this._bogyongName; }
			set { this._bogyongName = value; }
		}

		public String Dv
		{
			get { return this._dv; }
			set { this._dv = value; }
		}

		public String Banghyang
		{
			get { return this._banghyang; }
			set { this._banghyang = value; }
		}

		public String Buwi
		{
			get { return this._buwi; }
			set { this._buwi = value; }
		}

		public String Chiryo
		{
			get { return this._chiryo; }
			set { this._chiryo = value; }
		}

		public DRG0120U00Grd0120Item3Info() { }

		public DRG0120U00Grd0120Item3Info(String bunryu1Name, String bogyongCode, String bogyongName, String dv, String banghyang, String buwi, String chiryo)
		{
			this._bunryu1Name = bunryu1Name;
			this._bogyongCode = bogyongCode;
			this._bogyongName = bogyongName;
			this._dv = dv;
			this._banghyang = banghyang;
			this._buwi = buwi;
			this._chiryo = chiryo;
		}

	}
}

File: MED-CLOUD/med-data/src/main/java/nta/med/data/model/ihis/drug/DRG0120U00Grd0120Item3Info.java
package nta.med.data.model.ihis.drug;

public class DRG0120U00Grd0120Item3Info {
	private String bunryu1Name;  
    private String bogyongCode;                                              
    private String bogyongName;                                              
    private String dv;
    private String banghyang; 
    private String buwi;     
    private String chiryo;
	public DRG0120U00Grd0120Item3Info(String bunryu1Name, String bogyongCode,
			String bogyongName, String dv, String banghyang, String buwi,
			String chiryo) {
		super();
		this.bunryu1Name = bunryu1Name;
		this.bogyongCode = bogyongCode;
		this.bogyongName = bogyongName;
		this.dv = dv;
		this.banghyang = banghyang;
		this.buwi = buwi;
		this.chiryo = chiryo;
	}
	public String getBunryu1Name() {
		return bunryu1Name;
	}
	public void setBunryu1Name(String bunryu1Name) {
		this.bunryu1Name = bunryu1Name;
	}
	public String getBogyongCode() {
		return bogyongCode;
	}
	public void setBogyongCode(String bogyongCode) {
		this.bogyongCode = bogyongCode;
	}
	public String getBogyongName() {
		return bogyongName;
	}
	public void setBogyongName(String bogyongName) {
		this.bogyongName = bogyongName;
	}
	public String getDv() {
		return dv;
	}
	public void setDv(String dv) {
		this.dv = dv;
	}
	public String getBanghyang() {
		return banghyang;
	}
	public void setBanghyang(String banghyang) {
		this.banghyang = banghyang;
	}
	public String getBuwi() {
		return buwi;
	}
	public void setBuwi(String buwi) {
		this.buwi = buwi;
	}
	public String getChiryo() {
		return chiryo;
	}
	public void setChiryo(String chiryo) {
		this.chiryo = chiryo;
	}
}
